openapi/components/schemas/Common/CommonBillingPortal.yaml

```
@@ -19,6 +19,10 @@ properties:
     description: Billing portal website ID.
     allOf:
       - $ref: ../ResourceId.yaml
+  customDomain:
+    description: Custom domain.
```

nitpick: We should add more detail (if possible)

I'm assuming each custom domain relates to a websiteID

```suggestion
    description: Custom domain for the website ID.
```

---

It actually isn't for the website but it's a custom domain for the checkout form